### PR TITLE
Add separate methods for determining if a date is a working_day / holiday

### PR DESCRIPTION
**Pull request by ahjmorton**
Add separate methods for determining if a date is a `working_day?` / `holiday?`.

### CODE REVIEW

**Comment by dchambersgc** [comment]
Can we add a PR desc?

Looks good otherwise.

File: lib/business/calendar.rb [hunk 1]
```diff
@@ -63,10 +63,16 @@ def initialize(config)
     # non-weekend day) and not a holiday.
     def business_day?(date)
       date = date.to_date
-      return true if extra_working_dates.include?(date)
-      return false unless working_days.include?(date.strftime('%a').downcase)
-      return false if holidays.include?(date)
-      true
+      working_day?(date) && !holiday?(date)
```
Comment (ameykusurkar) [inline lib/business/calendar.rb:66]: This is so much cleaner 👍

File: README.md [hunk 2]
```diff
@@ -131,6 +131,20 @@ calendar.business_day?(Date.parse("Sunday, 8 June 2014"))
 # => false
 ```
 
+More specifically you can check if a given `business_day?` is either a `working_day?` or a `holiday?` using methods on `Business::Calendar`.
```
Comment (ameykusurkar) [inline README.md:134]: I'm wondering if people might expect `working_day?` to be an alias for `business_day?`, as sometimes we tend to use those terms interchangeably. Is it worth elaborating on it?

Comment (ahjmorton) [reply README.md:134]: Throughout the code and document it makes the distinction pretty clear e.g. in the yaml you specify "working_days" / "extra_working_days"

Comment (ahjmorton) [reply README.md:134]: e.g. the paragraph above

Comment (ameykusurkar) [reply README.md:134]: Fair enough 👍

**Pull request merged**